src/uwtools/resources/rocoto.jsonschema

```diff
@@ -130,6 +130,30 @@
           },
           "type": "object"
         },
+        "^sh(_.*)?$": {
+          "additionalProperties": false,
+          "properties": {
+            "attrs": {
+              "additionalProperties": false,
+              "properties": {
+                "runopt": {
+                  "type": "string"
+                },
+                "shell": {
+                  "type": "string"
+                }
+              },
```

Comment:
Pending the outcome of a discussion started in Slack today, I've provisionally left out support for the optional `name` attribute, under the assumption that we'd rather support naming via the key, e.g. `sh_foo` would lead to a `<sh name="foo">` XML tag. This may be consistent way to handle optional names, but may be an issue w.r.t. elements where order is desired (for example, from the Slack conversation, the `<metatask>` attribute `mode`, where the value `serial` calls for children to be processed serially and _in order_).